src/Zigzag_persistence/doc/Intro_zigzag_persistence.h

```diff
@@ -0,0 +1,69 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       Hannah Schreiber
+ *
+ *    Copyright (C) 2023 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+#ifndef DOC_ZIGZAG_PERSISTENCE_INTRO_ZIGZAG_PERSISTENCE_H_
+#define DOC_ZIGZAG_PERSISTENCE_INTRO_ZIGZAG_PERSISTENCE_H_
+
+// needs namespace for Doxygen to link on classes
+namespace Gudhi {
+namespace zigzag_persistence {
+
+/** \defgroup zigzag_persistence Zigzag Persistence
+ * @{
+ * \author    Cl&eacute;ment Maria, Hannah Schreiber
+ *
+ * We refer to the introduction page \ref persistent_cohomology for persistent (co)homology for an introduction
+ * to the topic.
+ * Zigzag persistence is a generalization of the latter. While standard persistence only allows to grow the filtered
+ * complex by adding simplices, zigzag persistence also allows removals. Hence the name "zigzag", as the module
+ * diagram will have arrows alterning between forward and backward.
+ *
+ * The module is partitioned in two types of classes: filtered and non-filtered.
+ * - There is one non-filtered class:
+ * @ref Zigzag_persistence. It computes the persistence by considering only the atomic operations in the filtration.
+ * If the order in which the operations are made still matters, the filtration values associated to an operation 
+ * is not token into account. For example, if a cycle is born at operation number 6 and dies at operation number 7, it
+ * will output a bar starting at 6 and ending at 7, even if both operations have the same filtration value and therefore
+ * the "real" bar has length 0.
+ * - There are two filtered classes: @ref Filtered_zigzag_persistence and @ref Filtered_zigzag_persistence_with_storage.
+ * They are both based on @ref Zigzag_persistence and manage additionnaly the filtration values which are ignored by 
+ * @ref Zigzag_persistence. They automatically translate the operation numbers into their corresponding filtration
+ * values and remove bars below a given length threshold. They also have more flexible inputs (the boundaries do not
```

Comment:
Where does this threshold appear?

Reply:
The threshold is given as an argument in `get_persistence_diagram` for `Filtered_zigzag_persistence_with_storage`. But for `Filtered_zigzag_persistence` the user has to handle it itself in the callback method. So I just removed the comment about it to avoid confusion.